tests/Behat/DrupalContext.php

```diff
@@ -201,4 +202,64 @@ protected function getLanguageIdByName(string $name): string {
     throw new \InvalidArgumentException("Language '{$name}' not found.");
   }
 
+  /**
+   * Redirect user on Node creation page.
+   *
+   * @param string $content_type_name
+   *   Content type name.
+   *
+   * @Given I am visiting the :content_type_name create node page
+   */
+  public function iAmVisitingTheCreateNodePage(string $content_type_name): void {
+    $node_bundle = $this->getEntityTypeByLabel($content_type_name);
+    $this->visitPath('node/add/' . $node_bundle);
+  }
+
+  /**
+   * Check that selectbox with node language selector is hidden.
+   *
+   * @param string $label
+   *   Label of selectbox.
+   *
+   * @Then I should not see selectbox with label :label
+   */
+  public function iShouldNotSeeSelectboxWithLabel(string $label): void {
+    $element = $this->getSession()
+      ->getPage()
+      ->findField($label);
+    if ($element) {
+      throw new \RuntimeException("Selectbox for selecting initial language still visible.");
+    }
+  }
+
+  /**
+   * Check that we have correct language for initial translation.
+   *
+   * @param string $title
+   *   Title of node.
+   *
+   * @Then I have to be sure that :title node translation only in site default language.
+   */
+  public function assertOnlyDefaultLanguageTranslationExist(string $title) {
+    $title_element = $this->getSession()
```

Comment:
Instead of relying on the session for this step, wouldn't it be better to use Drupal API to gather the translations of the node we are trying to check?